app/src/main/java/openfoodfacts/github/scrachx/openfood/views/ContinuousScanActivity.java

```diff
@@ -825,6 +831,18 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         }
     }
 
+    private boolean isNetworkAvailable() {
+        ConnectivityManager manager =
```

Comment:
please see:  isNetworkConnected in Utils.

Reply:
@teolemon I have made the changes. Kindly review :)

Reply:
@vanshikaarora  please, could you remove the unused import ?